src/Tasks/Microsoft.NET.Build.Tasks/CreateAppHost.cs

```diff
@@ -46,6 +46,8 @@ public class CreateAppHost : TaskBase
 
         public bool EnableMacOSCodeSign { get; set; } = false;
 
+        public bool DisableCetCompat { get; set; } = false;
```

Comment:
```suggestion
        public bool EnableCetCompat { get; set; } = true;
```

I really dislike the "Disable == true" semantics. I see the public facing one is simply `CetCompat`, which is very clear. I'd prefer if we try to keep "Enable == true" whenever possible.

Reply:
maybe just keep it neutral with `CetCompat`, since that is what its called in the proj files right?

Reply:
I like this phrasing because the default value for `bool` is `false`, so with this phrasing the default value for C# matches up with the default value for the switch. If we had decided to make it impossible to see the default value of a field in the type system I wouldn't care... but I've been disappointed so many times that I now just try to align the two views when possible.

Reply:
>  If we had decided to make it impossible to see the default value of a field in the type system I wouldn't care...

I'm fine leavin it, but using the semantics of the C# type system to dictate clear English semantics is a bit of a stretch. There is a long history of bugs related to "Disable == true" and avoiding that seems best when possible. The MSBuild property being `CetCompat` reduces this a great deal so users get to avoid this confusion.

Reply:
Yup I agree that I’ve seen bugs the other way as well. This was my personal opinion, I don’t think you should take it too seriously.